src/components/ChatList/elements/ChannelImage.tsx

```diff
@@ -74,7 +73,7 @@ const Big: React.FC<ChannelImageMainProps> = ({type, image, style}) => {
     );
   }
 
-  if (type === GROUP_INFO) {
+  if (type === 'GROUP_INFO') {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The comparison value for type has been changed from `GROUP_INFO` to 'GROUP_INFO'. This change might be incorrect if `GROUP_INFO` is a constant and not a string literal.
```diff
- if (type === GROUP_INFO) {
+ if (type === 'GROUP_INFO') {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->